09-appendix-backends.Rmd

```diff
@@ -36,65 +40,83 @@ If you are using the Dockstore CLI, [follow these instructions](https://docs.doc
+-   miniwdl's equivalent to `womtool` is `miniwdl check`, but it also includes `shellcheck` to check the command section of your tasks.
 
-* miniwdl supports call caching, but it is turned off by default.
+-   miniwdl supports call caching, but it is turned off by default.
 
-* miniwdl does not support the `gpu` or `disks` runtime attributes and will ignore them if present in a task's runtime section.
+-   miniwdl does not support the `gpu` or `disks` runtime attributes and will ignore them if present in a task's runtime section.
 
-* By default, miniwdl does not duplicate input files. If your workflow only needs to read input files, this helps save disk space, but if your workflow directly modifies input files, this can result in permission errors. A simple fix is to run miniwdl with the `--copy-input-files` flag.
+-   By default, miniwdl does not duplicate input files. If your workflow only needs to read input files, this helps save disk space, but if your workflow directly modifies input files, this can result in permission errors. A simple fix is to run miniwdl with the `--copy-input-files` flag.
 
 ## Backend-specific notes
 
 ### HPCs
+
 It is difficult to provide specific advice on HPCs, as they can vary greatly. Some general notes:
 
-* Some HPCs do not support the use of Docker due to security concerns
+-   Some HPCs do not support the use of Docker due to security concerns
 
-  * HPCs that do not allow the use of Docker may be able to run WDLs using alternative container technologies such as podman or rootless Docker
+    -   HPCs that do not allow the use of Docker may be able to run WDLs using alternative container technologies such as podman or rootless Docker
 
-* Some HPCs will use `disks` to determine which disk to run on, which can be useful for managing disk space
+-   Some HPCs will use `disks` to determine which disk to run on, which can be useful for managing disk space
 
 #### Fred Hutch HPC
-* The Fred Hutch HPC supports the use of Docker, so the `docker` runtime attribute works as expected
 
-* The `memory` and `gpu` runtime attributes are ignored, but `cpu` works as expected
+-   [PROOF](https://proof.fredhutch.org/) is a Shiny frontend that launches Cromwell Server and lets you manage your jobs from a GUI. User guide can be found [here](https://sciwiki.fredhutch.org/dasldemos/proof-how-to/).
+
+-   The Fred Hutch HPC supports the use of multiple JSON files going into the same workflow
+
+-   The Fred Hutch HPC supports the use of Docker, so the `docker` runtime attribute works as expected
+
+-   `memory` and `gpu` runtime attributes are ignored, but `cpu` works as expected. Here is [some information](https://sciwiki.fredhutch.org/scicomputing/compute_jobs/#memory) on approximating memory use.
+
+-   `walltime` attribute is a string ("HH:MM:SS") that specifies how much time you want to request for the task. Can also specify \>1 day, e.g. "1-12:00:00" is 1 day+12 hours.
```

Comment:
I'd recommend you elaborate on whether this is a hard or soft limit

Reply:
Not completely sure whether its a hard or soft limit - linked out to more info on walltime.